content/blog/managing-containers-on-aws-with-pulumi/index.md

```diff
@@ -328,4 +328,4 @@ As with the ECS example, run `pulumi up` to deploy the application. To check the
 
 ## Want to know more?
 
-Although these examples are simple, they demonstrate the basic building blocks for building, storing, and managing containers. They also show how to create ECS or EKS clusters for deploying apps. To get started with AWS and Pulumi check out the [AWS Guide]({{< relref "/docs/get-started/aws" >}}) for core services and the [Crosswalk for AWS guide]({{< relref "/docs/guides/crosswalk/aws" >}}) for convenience APIs that simplify deploying infrastructure as code. For a deeper dive into managing containers on AWS, check out our [How to Scale Your Amazon EKS Cluster: EC2, Managed Node Groups, and Fargate]({{< relref "/blog/aws-eks-managed-nodes-fargate" >}}).
+Although these examples are simple, they demonstrate the basic building blocks for building, storing, and managing containers. They also show how to create ECS or EKS clusters for deploying apps. To get started with AWS and Pulumi check out the [AWS Guide]({{< relref "/docs/get-started/aws" >}}) for core services and the [Crosswalk for AWS guide]({{< relref "/docs/guides/crosswalk/aws" >}}) for convenience APIs that simplify deploying infrastructure as code. For a deeper dive into managing containers on AWS, check out our blogs on [ECS vs Fargate vs EKS: The Lowdown on Containers in AWS](https://www.pulumi.com/blog/running-containers-in-aws-the-lowdown-ecs-fargate-and-eks/) and [How to Scale Your Amazon EKS Cluster: EC2, Managed Node Groups, and Fargate]({{< relref "/blog/aws-eks-managed-nodes-fargate" >}}).
```

Comment:
Do you want to add one more???

(IOW:  Although these examples are simple, they demonstrate the basic building blocks for building, storing, and managing AWS Containers. .... rather than what's there now.)

Reply:
spoke w. 97th and they DO want to take my suggested changes .. so you can add the edits and then merge .. thx